cot/src/config.rs

```diff
@@ -2045,6 +2063,26 @@ impl CacheUrl {
     pub fn as_str(&self) -> &str {
         self.0.as_str()
     }
+
+    /// Checks if the cache URL is for a Redis cache.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use cot::config::CacheUrl;
+    ///
+    /// let url = CacheUrl::from("redis://user:password@localhost:6379/0");
+    /// assert!(url.is_redis());
+    /// ```
+    #[must_use]
+    pub fn is_redis(&self) -> bool {
```

Comment:
I'm not sure if we should have this method here. I'd like to avoid referencing specific backends inside config.rs mostly so that we can more easily modularize subsystems such as Cache in the future.